cve_bin_tool/strings.py

```diff
@@ -46,7 +46,7 @@ def parse_strings(filename: str) -> str:
 
     if inpath("strings"):
         # use "strings" on system if available (for performance)
-        data = subprocess.check_output(["strings", filename])
+        data = subprocess.check_output(["strings", "-n 1", filename])
```

Comment:
```suggestion
        data = subprocess.check_output(["strings", "-n 3", filename])
```

Quick change to 3 as discussed.  We can probably open separate issues for the docs and native string implementation fixes, unless you want to add those to this PR.

Reply:
I applied the suggested change, I'll let you handle the rest in separate issues/PRs.